src/main/java/services/dataclasses/OntologyRaw.java

```diff
@@ -0,0 +1,11 @@
+package services.dataclasses;
+
+public class OntologyRaw {
```

Comment:
Should this class have a function like this?
```java
public OWLOntology toOWLOntology() {
  // return this object converted to an OWLOntology-object
}
```

Reply:
Added it!